docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -0,0 +1,115 @@
+.. _lora_finetune_recipe_label:
+
+=============================
+LoRA Single Device finetuning
+=============================
+
+This recipe supports finetuning on next-token prediction tasks using `LoRA <https://arxiv.org/abs/2106.09685>`_,
```

Comment:
This is a good start and I'd be very happy with just this intro right here. I think over time we could use this spot to provide more information about the core algorithm in the recipe and when a user would want to use it.

Reply:
Maybe we can take the current dos strings from recipes, their features, and put them here too. We could consider removing them from the recipes and the configs.